src/main/java/hudson/plugins/git/browser/ViewGitWeb.java

```diff
@@ -52,14 +56,14 @@ public URL getFileLink(Path path) throws IOException {
             String spec = buildCommitDiffSpec(url, path);
             return encodeURL(new URL(url, url.getPath() + spec));
         }
-        String spec = param(url).add("p=" + projectName).add("a=viewblob").add("h=" + path.getDst()).add("f=" +  path.getPath()).toString();
+        String spec = param(url).add("p=" + projectName).add("a=viewblob").add("h=" + path.getDst()).add("f=" + path.getPath()).toString();
         return encodeURL(new URL(url, url.getPath() + spec));
     }
 
-	private String buildCommitDiffSpec(URL url, Path path)
-			throws UnsupportedEncodingException {
-        return param(url).add("p=" + projectName).add("a=commitdiff").add("h=" + path.getChangeSet().getId()) + "#" +  URLEncoder.encode(path.getPath(),"UTF-8").toString();
-	}
+    private String buildCommitDiffSpec(URL url, Path path)
```

Comment:
Don't "fix" whitespace because it causes unnecessary conflicts in other pull requests.  Refer to the style guide in the contributing guideline for this repo.  In many other places it would be a good thing to correct the horribly inconsistent code formatting style.  In this particular plugin, it is not a good thing to correct the code formatting style.
```suggestion
    private String buildCommitDiffSpec(URL url, Path path)
```

Reply:
I apologize I've constantly ignored whitespace conflicts, I ran "Reformat Code" option available in IntelliJ which fixed all these whitespaces. 

I have tried to fix all of them in the latest commit, I hope I have reverted all these fixes.